package.json

```diff
@@ -5,11 +5,14 @@
   "main": "build/index.js",
   "scripts": {
     "build": "babel -s -d build examples",
-    "lint": "eslint examples && eslint test",
-    "test": "tape -r babel/register test/index.js",
+    "lint": "eslint examples/ test/ bin/",
+    "test": "tape -r babel-register test/index.js",
     "watch": "watch 'clear && npm run -s lint && npm -s test' test/ examples/",
     "prepublish": "npm run lint && npm run test && npm run build"
   },
+  "bin": {
+    "check-compose": "./bin/check-compose.js"
```

Comment:
I tried that. Spent about an hour. Doesn't work unfortunately. The property value must be a file. It can't be a shell command.
Moreover, this can't be debugged with iron-node. Whereas, this PR makes sure the tests can be debuggable.